packages/e2e-tests/src/specs/editor/publishingFlow.js

```diff
@@ -30,27 +29,6 @@ import {
 // Disable for https://github.com/google/web-stories-wp/issues/6238
 // eslint-disable-next-line jest/no-disabled-tests
 describe.skip('Publishing Flow', () => {
-  let stopRequestInterception;
-
-  beforeAll(async () => {
-    await page.setRequestInterception(true);
-    stopRequestInterception = addRequestInterception((request) => {
-      if (request.url().startsWith('https://cdn.ampproject.org/')) {
-        request.respond({
-          status: 200,
-          body: '',
-        });
-      } else {
-        request.continue();
-      }
-    });
-  });
-
-  afterAll(async () => {
-    await page.setRequestInterception(false);
-    stopRequestInterception();
-  });
```

Comment:
Removing so we actually send requests to the AMP CDN?

Reply:
Yes.

a) because there was no real reason for this request interception the first place + we get more accurate results
b) because FF doesn't support request interception